layers/meta-balena-beaglebone/recipes-bsp/u-boot/u-boot-ti-staging_%.bbappend

```diff
@@ -1,10 +1,33 @@
 UBOOT_KCONFIG_SUPPORT = "1"
 inherit resin-u-boot
 
+UBOOT_MACHINE = "am335x_evm_config"
+UBOOT_MACHINE:beagleplay = "am62x_evm_a53_defconfig"
+UBOOT_MACHINE:beagleplay-k3r5 = "am62x_evm_r5_defconfig"
+
+# Use BeagleBone's u-boot until u-boot-ti-staging has the BBAI64 support upstreamed.
+BRANCH:beagleplay= "v2023.04-ti-09.00.00.010-BeaglePlay"
+BRANCH:beagleplay-k3r5= "v2023.04-ti-09.00.00.010-BeaglePlay"
+UBOOT_GIT_URI:beagleplay = "git://git.beagleboard.org/beagleboard/u-boot.git"
+UBOOT_GIT_URI:beagleplay-k3r5 = "git://git.beagleboard.org/beagleboard/u-boot.git"
+SRCREV:beagleplay = "4a4f4d88ffb620a3d389900f648abb24008f3ddc"
```

Comment:
Once you make the changes requested in https://github.com/balena-os/balena-beaglebone/pull/666/commits/7b5f976b5176be7c8dfd8c114e971c43bc13aa44 you should also be able to remove the aboves related to u-boot switching

Reply:
See my comments above. We still need this branch (as opposed to the ti mainline).

Reply:
@delhiryder what you did here was you switched the git url of the u-boot-ti-staging recipe to point to git://git.beagleboard.org/beagleboard/u-boot.git

But what I was saying is that the u-boot-bb.org recipe in meta-ti is the one that's using this u-boot source: https://git.yoctoproject.org/meta-ti/tree/meta-ti-bsp/recipes-bsp/u-boot/u-boot-bb.org_git.bb?h=kirkstone